src/utils/formatTime.ts

```diff
@@ -3,7 +3,13 @@ import { BigNumber, BigNumberish } from '@ethersproject/bignumber'
 import { SECONDS_IN_DAY, SECONDS_IN_HOUR } from 'constants/numbers'
 import { DurationUnitsOption } from 'constants/time'
 
-export function detailedTimeString(timeSeconds?: BigNumberish) {
+export function detailedTimeString({
```

Comment:
would love to have some tests for this 😏

Reply:
WDYM?

Reply:
@johnnyd-eth added a quickie: https://github.com/jbx-protocol/juice-interface/pull/822/commits/f33d1c44b19fc22c02c39ce91b92a27644cb88b6

These utils are usually pretty easy to write tests for and given they are pretty long lasting (i.e. don't get changed often), it can be valuable to write tests for them.